### ISSUE TITLE
How to download the csv2rdf-standalone.jar ?

### ISSUE THREAD

**Issue by vemonet** [Swirrl/csv2rdf#52]
Hi, I have looked into the `README.md` and `doc/` folder docs but I could not find how to download or build the executable `csv2rdf-standalone.jar`

The ideal would be to be able to download the `csv2rdf-standalone.jar` directly from the GitHub releases (if you don't have already another distribution system). You can easily attach any file, like a jar file, to a new (or existing) release

This would allow users to easily download the latest stable release of the executable jar file to run it directly with Java (without the need for clojure, knowledge or dependencies)

Thanks for this library, it looks really interesting!

**Comment by lkitching** [comment]
Hi, I've updated the main documentation to add a section on building uberjars from source. The github releases should contain the standalone jars but since we build them from source we haven't noticed this oversight so thanks for raising this issue. There's an outstanding issue #14 to add these automatically for new versions but for now I've manually added `csv2rdf-standalone.jar` to [the latest release](https://github.com/Swirrl/csv2rdf/releases/tag/0.4.4).

Let me know if you have any problems.
